node/express.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const lambdaExpress = require('aws-serverless-express');
```

Comment:
Can we name this `awsServerlessExpress` to be consistent with the awslabs docs? I suspect people will have copy/pasted things so keeping them consistent would be :+1: